### PR TITLE
fish sounds

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
sounds fishy around here
<details>

https://github.com/user-attachments/assets/9131a1c5-a3ad-4fa8-b858-9c3ddba3d450

https://github.com/user-attachments/assets/0d7381a8-b0c3-4796-befa-b71e4819aaa8

https://github.com/user-attachments/assets/a714c857-11bd-42b4-adaf-7447c236711e

https://github.com/user-attachments/assets/f8697933-1e5f-4403-b3f2-959e6e0c8254

https://github.com/user-attachments/assets/72530952-14ea-4c97-aa32-76c80036fff5

</details>

## Why It's Good For The Game

image

## Changelog

:cl: grungussuss
sound: fish now have new sounds
/:cl:

### CODE REVIEW

**Review by Ghommie — APPROVED** [review]

File: code/modules/fishing/fish/_fish.dm [hunk 1]
```diff
@@ -310,6 +313,9 @@
 
 	SEND_SIGNAL(src, COMSIG_FISH_FORCE_UPDATED, weight_rank, bonus_malus)
 
+	if(force >=15 && hitsound == SFX_DEFAULT_FISH_SLAP) // don't override special attack sounds
+		hitsound = 'sound/creatures/fish/fish_slap2.ogg' // do more damage - do heavier slap sound
```
Comment (Ghommie) [inline code/modules/fishing/fish/_fish.dm:317]: Oh, I've forgot, but you need to also do the opposite at the start of update_fish_force() (and perhaps use a sfx define for it too).

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/fishing/fish/_fish.dm (hunk 1 above)
Comment (Sadboysuss) [reply code/modules/fishing/fish/_fish.dm:317]: oof

File: code/modules/fishing/fish/_fish.dm [hunk 2]
```diff
@@ -310,6 +315,9 @@
 
 	SEND_SIGNAL(src, COMSIG_FISH_FORCE_UPDATED, weight_rank, bonus_malus)
 
+	if(force >=15 && hitsound == SFX_DEFAULT_FISH_SLAP) // don't override special attack sounds
+		hitsound = SFX_DEFAULT_FISH_SLAP // do more damage - do heavier slap sound
```
Comment (Ghommie) [inline code/modules/fishing/fish/_fish.dm:319]:
```suggestion
		hitsound = SFX_ALT_FISH_SLAP // do more damage - do heavier slap sound
```

File: code/modules/fishing/fish/_fish.dm [hunk 3]
```diff
@@ -282,6 +285,8 @@
 
 ///Reset weapon-related variables of this items and recalculates those values based on the fish weight and size.
 /obj/item/fish/proc/update_fish_force()
+	if(force <= 15 && hitsound == SFX_ALT_FISH_SLAP)
```
Comment (Ghommie) [inline code/modules/fishing/fish/_fish.dm:288]:
```suggestion
	if(force >= 15 && hitsound == SFX_ALT_FISH_SLAP)
```

**Review by Ghommie — APPROVED** [review]

**Pull request merged**